large-file-uploader-app/assets/iframe.html

```diff
@@ -21,7 +21,7 @@
 					function(data) {
 						var accountEntryId = data['ticket.customField:custom_field_360013377572']
 
-						if( accountEntryId !== null && accountEntryId !== '' ) {
+						if( accountEntryId && accountEntryId !== '' ) {
 							client.get('ticket.id').then(
```

Comment:
Do we want to `.catch` anything?

Reply:
Yeah, we should. Not sure if we need an `else` though. :thinking:

Reply:
Nah we don't have to have an `else` every time.

Reply:
I know. I am not sure though if we need to display any placeholder, like "You can't upload large files, because this ticket does not have an account, you dummy". Or something like that.

Reply:
Ah, I see what you mean.